RybakAndriy/HW5/Task2.py

```diff
@@ -0,0 +1,8 @@
+numbers_fib = int(input("entered number n: "))
+
```

Comment:
good job
look at this code
n = int(input('enter some number from fibonacci list= '))

fibonacci_numbers = [0, 1]

for i in range(2,n):
        fibonacci_numbers.append(fibonacci_numbers[i-1]+fibonacci_numbers[i-2])